src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -106,6 +106,16 @@ public class MonoAOTCompiler : Microsoft.Build.Utilities.Task
     /// </summary>
     public bool UseDirectPInvoke { get; set; }
 
+    /// <summary>
+    /// PInvokes to call directly.
+    /// </summary>
+    public ITaskItem[] DirectPInvokes { get; set; } = Array.Empty<ITaskItem>();
+
+    /// <summary>
+    /// File with list of PInvokes to call directly.
```

Comment:
This is a bit nit picking, but isn't it suppose to be `List of files with list of...`. 
Are we planning to provide some documentation on how to use the newly added parameters for the MSBuild task, but also to the AOT compiler (I know that we have flags undocumented but would still improve user experience)

Reply:
You're right, thanks for the catch :). I think its a good idea to have some docs to elaborate on how to leverage the parameters and what effect will happen if they are provided. I'll try to add some more in the summaries and this PR's description after doing some more testing. Is there some other place we have in mind to elaborate on different usage scenarios of the mono aot compiler and how the parameters should be used/set?

Reply:
I believe that documentation comments in aot-compiler would be good.

Reply:
Added more documentation to the options in aot-compiler.c, added more documentation to the summary in MonoAOTCompiler.cs